adoc/admin-troubleshooting-etcd.adoc

```diff
@@ -0,0 +1,136 @@
+= ETCD Troubleshooting
+
+== Introduction
+
+This document aims to describe debugging a {etcd} cluster.
+
+The required etcd logs are part of the `supportconfig`, a utility that collects are the required information for debugging a problem. The rest of the document provides information on how you can obtain these information manually.
+
+== ETCD container
+
+ETCD is a distributed reliable key-value store for the most critical data of a distributed system. It is running **only on the master** nodes in a form a container application. For instance, in a cluster with 3 master nodes, it is expected
+to have 3 etcd instances as well:
+
+[source,bash]
+----
+kubectl get pods -n kube-system -l component=etcdNAME
+                                READY   STATUS    RESTARTS   AGE
+etcd-vm072044.qa.prv.suse.net   1/1     Running   1          7d
+etcd-vm072050.qa.prv.suse.net   1/1     Running   1          7d
+etcd-vm073033.qa.prv.suse.net   1/1     Running   1          7d
+----
+
+The specific configuration which `etcd` is using to start, is the following:
+
+[source,bash]
+----
+etcd \
+      --advertise-client-urls=https://<YourMasterNodeIP>:2379 \
+      --cert-file=/etc/kubernetes/pki/etcd/server.crt  \
+      --client-cert-auth=true --data-dir=/var/lib/etcd \
+      --initial-advertise-peer-urls=https://<YourMasterNodeIP>:2380 \
+      --initial-cluster=vm072050.qa.prv.suse.net=https://<YourMasterNodeIP>:2380 \
+      --key-file=/etc/kubernetes/pki/etcd/server.key \
+      --listen-client-urls=https://127.0.0.1:2379,https://<YourMasterNodeIP>:2379 \
+      --listen-peer-urls=https://<YourMasterNodeIP>:2380 \
+      --name=vm072050.qa.prv.suse.net \
+      --peer-cert-file=/etc/kubernetes/pki/etcd/peer.crt \
+      --peer-client-cert-auth=true \
+      --peer-key-file=/etc/kubernetes/pki/etcd/peer.key \
+      --peer-trusted-ca-file=/etc/kubernetes/pki/etcd/ca.crt \
+      --snapshot-count=10000 --trusted-ca-file=/etc/kubernetes/pki/etcd/ca.crt
+----
+
+[NOTE]
+====
+For more information related to ETCD, we **highly** recommend you to read https://etcd.io/docs/v3.4.0/faq/[ETCD FAQ] page.
+====
+
+== logging
+
+Since `etcd` is running in a container, that means it is no more controlled by `systemd`, thus any commands related to that (e.g. `journalctl`) will fail, therefore you need to use container debugging approach instead.
```

Comment:
```suggestion
Since `etcd` is running in a container, that means it is not controlled by `systemd`, thus any commands related to that (e.g. `journalctl`) will fail, therefore you need to use container debugging approach instead.
```

Reply:
Thanks